service-python/pdfgenerator/src/lib/templates/hypertension-v2.html

```diff
@@ -271,7 +271,22 @@ <h2 class="m-0 mb-13"><b>Schedular evidence for hypertension – blood pressure
             <td {% if bp_reading.organization == "" and bp_reading.dataSource == "MAS"  %}class="empty" {% endif %}>{% if not bp_reading.organization and bp_reading.dataSource == "MAS" %}n.a.{% elif not bp_reading.organization and bp_reading.dataSource == "LH" %} VAMC record {% elif bp_reading.organization and bp_reading.dataSource == "LH" %} VAMC record - {{ bp_reading.organization }}{% else %}{{ bp_reading.organization }}{% endif %}</td>
             <td {% if bp_reading.receiptDate == "" %}class="empty" {% endif %}>{% if bp_reading.receiptDate == ""%}n.a.{% else %}{{ bp_reading.receiptDate }}{% endif %}</td>
             <td {% if bp_reading.page == "" %}class="empty" {% endif %}>{% if bp_reading.page == ""%}n.a.{% else %}{{ bp_reading.page }}{% endif %}</td>
-            <td {% if bp_reading.document == "" %}class="empty" {% endif %}>{% if bp_reading.document == ""%}n.a.  - see VistA/Capri to verify{% else %}{{ bp_reading.document }}{% endif %}</td>
+            <td {% if bp_reading.document == "" %}class="empty" {% endif %}>
+              {% if bp_reading.dataSource == "MAS" %}
```

Comment:
```suggestion
```

Reply:
I think the logic could be simplified if this `if` statement is removed. Then there would be no need for the else block because it would be handled the same by the `elif` and `else` in this block.

Reply:
After Afsin's text update request, now the conditional branches are different so I'll leave it for now. I'll remove if there's another update that make the both branches the same again